Updating the Note in Viewing conditional updates in the web console (#78826)
openshift/openshift-docs: modules/update-conditional-web-console.adoc

[NOTE]
====
If a version with known issues is selected, more information is provided with potential risks that are associated with the version.

> If the `Include versions with known issues` feature is disabled and only Recommended versions are displayed, that means no upgrade `versions with known issues` are available.

Author: That conveys the meaning much better, thanks @opayne1

> If the `Include versions with known issues` feature is disabled and only Recommended versions are displayed, that means no upgrade `versions with known issues` are available.

Reviewer: <img width="600" alt="Screenshot 2024-07-17 at 11 09 33 AM" src="https://github.com/user-attachments/assets/83e34751-0625-4f6b-9bb9-4270299aed51">

> If the `Include versions with known issues` feature is disabled and only Recommended versions are displayed, that means no upgrade `versions with known issues` are available.

Reviewer: ```suggestion
If the `Include versions with known issues` feature is disabled, only recommended versions are displayed. When disabled, versions `Have known issues` are not displayed.
```
